### PR TITLE
Update release notes - 3

### PR DESCRIPTION
**Pull request by NikhilBartwal**
Issue Reference: #2463 
`description='release notes'` field has been removed and `RELEASE_NOTES` has been added to 5 datasets.

### CODE REVIEW

File: tensorflow_datasets/image/lsun.py [hunk 1]
```diff
@@ -72,8 +72,9 @@ class Lsun(tfds.core.GeneratorBasedBuilder):
           name=category,
           description="Images of category %s" % category,
           version=tfds.core.Version(
-              "3.0.0",
-              "New split API (https://tensorflow.org/datasets/splits)"),
+              "3.0.0"),
```
Comment (Conchylicultor) [inline tensorflow_datasets/image/lsun.py:75]: Fix formatting ?

File: tensorflow_datasets/image/shapes3d.py [hunk 2]
```diff
@@ -56,7 +56,10 @@ class Shapes3d(tfds.core.GeneratorBasedBuilder):
   """Shapes3d data set."""
 
   VERSION = tfds.core.Version(
-      "2.0.0", "New split API (https://tensorflow.org/datasets/splits)")
+      "2.0.0")
```
Comment (Conchylicultor) [inline tensorflow_datasets/image/shapes3d.py:59]: Same here

**Review by Conchylicultor — CHANGES_REQUESTED** [review]
Thank you!

File: tensorflow_datasets/image_classification/bigearthnet.py [hunk 3]
```diff
@@ -114,7 +114,12 @@ def __init__(self, selection=None, **kwargs):
       raise ValueError('selection must be one of %s' % _DATA_OPTIONS)
 
     v100 = tfds.core.Version(
-        '1.0.0', 'New split API (https://tensorflow.org/datasets/splits)')
+        '1.0.0')
```
Comment (Conchylicultor) [inline tensorflow_datasets/image_classification/bigearthnet.py:117]: The review from a PR should apply to all PR. In this case the formatting isn't correct.

**Review by Conchylicultor — APPROVED** [review]
Thank you.

File: tensorflow_datasets/image_classification/bigearthnet.py (hunk 3 above)
Comment (NikhilBartwal) [reply tensorflow_datasets/image_classification/bigearthnet.py:117]: @Conchylicultor Apologies for that! Should I open another PR for fixing the formatting as this has been merged?

**Pull request merged**